src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,17 @@
+.footer {
+  padding: 0;
+  margin: 0;
```

Comment:
If you want to reset margins, do it in `reset.scss` file.
According to BEM we cannot give outer spacings to BEM-blocks (`.footer` is a BEM-block)